axlearn/experiments/text/gpt/fuji.py

```diff
@@ -67,7 +67,7 @@ class Version(enum.Enum):
 VOCAB_SIZE = {
     Version.V1: 32 * 1024,
     Version.V2: 32 * 1024,
-    Version.V3: 128256,
+    Version.V3: 128 * 1024,
```

Comment:
llama 3 is supposed to be 128256. Are there any plans to stick to standard llama 3?

Reply:
I add -tiktoken configs and they will use 128256.
the configs without -tiktoken will stay unchanged.